File: 프로그래머스/0/120841. 점의 위치 구하기/점의 위치 구하기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int dot[], size_t dot_len) {
    int answer = 0;
    
    if(dot[0] != 0 && dot[1] != 0){
            if(dot[0] > 0 ){
                answer = dot[1] > 0 ? 1 : 4;
            }
            else{
                answer = dot[1] > 0 ? 2 : 3;
            }
        }
    
    return answer;
}

File: 프로그래머스/0/120841. 점의 위치 구하기/점의 위치 구하기.py
def solution(dot):
    answer = 0
    
    if dot[0] != 0 and dot[1] != 0:
            if dot[0] > 0:
                answer = dot[1] > 0 and 1 or 4
            else:
                answer = dot[1] > 0 and 2 or 3
                
    return answer